bin/dfx-ii-create-dummy-anchor

```diff
@@ -31,12 +31,24 @@ trap clean_up EXIT
 dfx identity export "$DFX_IDENTITY" | openssl ec -out "$IDENTITY_PUB_KEY_FILE" -pubout -outform der 2>/dev/null
 
```

Comment:
Funny thing:  The path to the candid is specified in dfx.json, yet not picked up.  If I specify the path explicitly, fields are decoded correctly.